crates/primitives-traits/src/block/body.rs

```diff
@@ -53,15 +60,15 @@ pub trait BlockBody:
 
     /// Calculate the withdrawals root for the block body, if withdrawals exist. If there are no
     /// withdrawals, this will return `None`.
-    fn calculate_withdrawals_root(&self) -> Option<B256> {
-        Some(proofs::calculate_withdrawals_root(self.withdrawals()?))
-    }
+    // todo: can be default impl if `calculate_withdrawals_root` made into a method on 
+    // `Withdrawals` and `Withdrawals` moved to alloy
+    fn calculate_withdrawals_root(&self) -> Option<B256>;
```

Comment:
no big deal rn, we're also want to move some proofs functions to alloy-trie so they can be reused